### PR TITLE
fix urlUtils.getProxyUrl always add the trailing slash (close #342)

### LINKED ISSUE #342: urlUtils.getProxyUrl always add the trailing slash
For example:
urlUtils.getProxyUrl('http://google.com') -> "http://<host>:<port>/<sessionId>/http://google.com/"

Reproduced in IE, Chrome, FF

### PR DESCRIPTION
**Pull request by miherlosev**
/cc @churkin @LavrovArtem

### CODE REVIEW

File: src/client/utils/destination-location.js [hunk 1]
```diff
@@ -66,7 +67,13 @@ export function resolveUrl (url, doc) {
             return resolveUrl(url, window.parent.document);
     }
 
-    return urlResolver.href;
+    var result           = urlResolver.href;
+    var hasTrailingSlash = endsWith(url, '/');
```
Comment (LavrovArtem) [inline src/client/utils/destination-location.js]: may be use
```javascript
url.lastIndexOf('/') === url.length - 1
```

**Comment by miherlosev** [comment]
`endsWith` and `trim` group into `utils/string`.
I use the @LavrovArtem variant for `endsWith` implementation.
It is breaking change.
FPR

**Comment by churkin** [comment]
lgtm

**Comment by miherlosev** [comment]
for `endsWith` use imprlementation from http://stackoverflow.com/questions/280634/endswith-in-javascript
FPR

**Comment by LavrovArtem** [comment]
lgtm

**Pull request merged**